public/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html ng-app="AlwaysDo">
```

Comment:
I'm rather tempted to put the `ng-app` on the body, rather than the HTML - that just gives us options when we might want to include multiple Angular apps later on.